packages/muelu/src/Graph/StructuredAggregation/MueLu_StructuredAggregationFactory_decl.hpp

```diff
@@ -56,6 +56,7 @@
 #include "MueLu_StructuredAggregationFactory_fwd.hpp"
 #include "MueLu_Level_fwd.hpp"
 #include "MueLu_Exceptions.hpp"
+#include "MueLu_AggregationStructuredAlgorithm_fwd.hpp"
```

Comment:
Why is this needed here when it's not referenced anywhere in the file? I would have assumed that it would go in the `_def` file instead?

Reply:
I believe this is needed to make short names work.